### PR TITLE
feat(guide): Recommended naming scheme - Add Enable Rename instructions

### PR DESCRIPTION
**Pull request by TRaSH-**
# Pull Request

## Purpose

Add Enable Rename instructions

## Open Questions and Pre-Merge TODOs

Some people didn't see the episode or movie naming options because they didn't know they had to enable rename.

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/Radarr/Radarr-recommended-naming-scheme.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ Once you clicked on the button it should look like this and you should see all t
 
 ![Unhide Advanced](images/unhide-advanced.png)
 
+And as last enable Rename Movies to reveal the movies renaming options.
```
Comment (yammes08) [inline docs/Radarr/Radarr-recommended-naming-scheme.md:26]:
```suggestion
Then, enable Rename Movies to reveal the movies renaming options.
```

File: docs/Sonarr/Sonarr-recommended-naming-scheme.md [hunk 2]
```diff
@@ -23,6 +23,10 @@ Once you clicked on the button it should look like this and you should see all t
 
 ![Unhide Advanced](images/unhide-advanced.png)
 
+And as last enable Rename Episodes to reveal the episode renaming options.
```
Comment (yammes08) [inline docs/Sonarr/Sonarr-recommended-naming-scheme.md:26]:
```suggestion
Then, enable Rename Episodes to reveal the episode renaming options.
```

**Review by yammes08 — CHANGES_REQUESTED** [review]
A couple of language tweaks, but otherwise all good.

File: docs/Radarr/Radarr-recommended-naming-scheme.md (hunk 1 above)
Comment (zakkarry) [inline docs/Radarr/Radarr-recommended-naming-scheme.md:26]:
```suggestion
Finally, enable `Rename Movies` to reveal the movie renaming options.

File: docs/Sonarr/Sonarr-recommended-naming-scheme.md (hunk 2 above)
Comment (zakkarry) [inline docs/Sonarr/Sonarr-recommended-naming-scheme.md:26]:
```suggestion
Finally, enable `Rename Episodes` to reveal the episode renaming options.

**Review by zakkarry — CHANGES_REQUESTED** [review]
language stuff...

**Comment by TRaSH-** [comment]
so for which one are we going ?

**Comment by yammes08** [comment]
> so for which one are we going ?

Go for zakkarry's - that way _he_ can owe _me_ a drink!

**Comment by zakkarry** [comment]
> > so for which one are we going ?
> 
> Go for zakkarry's - that way _he_ can owe _me_ a drink!

lol damn!

**Comment by TRaSH-** [comment]
> > so for which one are we going ?
> 
> Go for zakkarry's - that way _he_ can owe _me_ a drink!

done

**Review by zakkarry — APPROVED** [review]

**Review by nuxencs — APPROVED** [review]
lgtm

**Review by yammes08 — APPROVED** [review]

**Pull request merged**